axelrod/tests/unit/test_sequence_player.py

```diff
@@ -0,0 +1,84 @@
+"""Test for the Thue-Morse strategies."""
+import unittest
+
+import axelrod
+from .test_player import TestPlayer, TestOpponent
+from axelrod.strategies.sequence_player import recursive_thue_morse, SequencePlayer
+
+
+C, D = axelrod.Actions.C, axelrod.Actions.D
+
+class TestThueMoreGenerator(unittest.TestCase):
+    def test_sequence(self):
+        expected = [0, 1, 1, 0, 1, 0, 0, 1, 1, 0, 0, 1, 0, 1, 1, 0]
+        for i, e in enumerate(expected):
+            self.assertEqual(recursive_thue_morse(i), e)
+
+
+class TestSequencePlayer(unittest.TestCase):
+    def test_sequence_player(self):
+        """Basic test for SequencePlayer."""
+        def cooperate_gen():
+            yield C
+        player = SequencePlayer(cooperate_gen)
+        opponent = TestOpponent()
+        self.assertEqual(C, player.strategy(opponent))
+
+
+class TestThueMorse(TestPlayer):
+
+    name = 'ThueMorse'
+    player = axelrod.ThueMorse
+    expected_classifier = {
+        'memory_depth': float('inf'),
+        'stochastic': False,
+        'makes_use_of': set(),
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def test_strategy(self):
+        """Test that strategy is randomly picked (not affected by history)."""
```

Comment:
This docstring needs to change. Could you make it (and the test) similar to lines 71 and 73.